merkle/merkle_hasher.go

```diff
@@ -113,3 +123,127 @@ func (self TreeHasher) _hash_fold(hashes []common.Uint256) common.Uint256 {
 
 	return accum
 }
+
+func HashLeaf(data []byte) common.Uint256 {
+	tmp := append([]byte{0}, data...)
+	return sha256.Sum256(tmp)
+}
+
+func HashChildren(left, right common.Uint256) common.Uint256 {
+	data := append([]byte{1}, left[:]...)
+	data = append(data, right[:]...)
+	return sha256.Sum256(data)
+}
+
+func MerkleLeafPath(data []byte, hashes []common.Uint256) ([]byte, error) {
+	buf := bytes.NewBuffer(make([]byte, 0, len(hashes)*common.UINT256_SIZE+len(data)))
+	index := getIndex(HashLeaf(data), hashes)
+	if index < 0 {
+		return nil, fmt.Errorf("%s", "values doesn't exist!")
+	}
+	d := depth(len(hashes))
+	merkleTree := MerkleHashes(hashes, d)
+	if err := serialization.WriteUint64(buf, uint64(len(data))); err != nil {
```

Comment:
`serialization` package is deprecated and will be removed soon, replace all usage with `ZeroCopySink` instead.

Reply:
it had been removed

Reply:
need remove all usage in this pr.